functions/HttpTriggerAPIUsersIdTaskId/__init__.py

```diff
@@ -6,71 +10,124 @@
 #GET API method function
 def get(param1, param2):
     #connects to db
+    logging.debug('Attempting a db conenction')

```

Comment:
I feel weird commenting about spelling, but connection

Reply:
sure, it's all fair game